nkms/policy/models.py

```diff
@@ -1,19 +1,20 @@
 import asyncio
 import binascii
-import maya
 
+import maya
 import msgpack
-from npre.constants import UNKNOWN_KFRAG
 
 from nkms.characters import Alice, Bob, Ursula
-from nkms.crypto import api
 from nkms.crypto.api import keccak_digest
-from nkms.crypto.constants import NOT_SIGNED, HASH_DIGEST_LENGTH
-from nkms.crypto.fragments import KFrag, PFrag
-from nkms.crypto.powers import EncryptingPower, SigningPower
+from nkms.crypto.constants import NOT_SIGNED, KECCAK_DIGEST_LENGTH, \
+    PUBLIC_KEY_LENGTH, UNKNOWN_KFRAG
+from nkms.crypto.powers import SigningPower
 from nkms.crypto.signature import Signature
+from nkms.crypto.splitters import key_splitter
 from nkms.crypto.utils import BytestringSplitter
-from nkms.keystore.keypairs import PublicKey
+from nkms.network.constants import BYTESTRING_IS_TREASURE_MAP
+from umbral.keys import UmbralPublicKey
+from umbral.umbral import Capsule
 
 
 class Contract(object):
```

Comment:
Not to be confused with "smart contract"...

Reply:
Hold my beer.

Reply:
"Not smart contract" :-D

Reply:
DumbContract

Reply:
✔️